src/Sulu/Bundle/AdminBundle/Admin/View/TabViewChildBuilderTrait.php

```diff
@@ -33,6 +33,11 @@ private function setTabPriorityToView(View $view, int $tabPriority): void
         $view->setOption('tabPriority', $tabPriority);
     }
 
+    private function setTabGapToView(View $view, bool $enableTabGap): void
+    {
+        $view->setOption('tabGap', $enableTabGap);
```

Comment:
we used `searchable` or `paginated` in other places. therefore i would use something like `hasTabGap` to indicate that it is a boolean 🙂

Reply:
i think the default case is that there is a tab gap, right? then i would call it `disableTabGap` to make clear that there will be a tab gap if the option is not set 🙂